google-cloud-bigquery/src/main/scala/akka/stream/alpakka/googlecloud/bigquery/InsertAllRetryPolicy.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.googlecloud.bigquery
+
+sealed abstract class InsertAllRetryPolicy {
+  def retry: Boolean
+  def deduplicate: Boolean
+}
+
+case object NoRetries extends InsertAllRetryPolicy {
```

Comment:
I would move these to an `InsertAllRetryPolicy` object too.

Reply:
Actually I agree here! `InsertAllRetryPolicy` is basically an enum.

Reply:
I ended up reverting this. Still not good at Scala/Java interop but I had problems accessing from Java when they were nested inside.